app/design/frontend/UnitedArtsEducation/unitednow/Magento_Customer/web/css/source/_extend.less

```diff
@@ -16,6 +16,12 @@
                 padding: 0 4px;
             }
         }
+
+        .message{
+            &.success{
+                margin-top:25px;
+            }
+        }
```

Comment:
This doesn't actually fix the underlying issue. The message may not be a success message, or there may not be a message at all. The problem is with `.block-collapsible-nav`, which has absolute positioning against `.page-main`. I suggest adding more padding to the top of `.page-main` to account for this, so that no matter what the first thing inside of it is, it will still be out of the way of the dropdown.